client/galaxy/scripts/mvc/tool/tool-form-base.js

```diff
@@ -79,11 +79,6 @@ export default FormBase.extend({
                 self.deferred.reset();
                 self.deferred.execute(process => {
                     self.model.get("postchange")(process, self);
-                    if (self.model.get("listen_to_history")) {
```

Comment:
Galaxy is now unused, and the linter doesn't like that (https://circleci.com/gh/galaxyproject/galaxy/15416?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link)